Source/WebCore/page/scrolling/mac/ScrollerPairMac.h

```diff
@@ -94,6 +94,9 @@ class ScrollerPairMac : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<S
     ScrollerMac& verticalScroller() { return m_verticalScroller; }
     ScrollerMac& horizontalScroller() { return m_horizontalScroller; }
 
+    CheckedRef<ScrollerMac> checkedVerticalScroller() { return m_verticalScroller; }
+    CheckedRef<ScrollerMac> checkedHorizontalScroller() { return m_horizontalScroller; }
```

Comment:
```suggestion
    ScrollerMac& verticalScroller() { return m_verticalScroller; }
    CheckedRef<ScrollerMac> checkedVerticalScroller() { return m_verticalScroller; }
    ScrollerMac& horizontalScroller() { return m_horizontalScroller; }
    CheckedRef<ScrollerMac> checkedHorizontalScroller() { return m_horizontalScroller; }
```
This makes it clearer the comment still applies.